File: Programming Language(ver.2)/Go/708.go
package main
import "fmt"

func main(){
	var count int = 0;
	for i := 0; i < 3; i++{
		var hour, minutes int
		fmt.Scan(&hour)
		fmt.Scan(&minutes)
		var conv_hour int = hour
		var conv_minutes int = minutes
		if hour == 0{
			conv_hour = 12
		} else if hour > 12 {
			conv_hour = hour - 12
		}
		if hour < 12 {
			fmt.Printf("AM %d:%d\n", conv_hour, conv_minutes)
			count += 1
		}else{
			fmt.Printf("PM %d:%d\n", conv_hour, conv_minutes)
		}
	}
	fmt.Println(count)
}

File: Programming Language(ver.2)/PHP/708.php
<?php
    $count = 0;
    for($i = 0; $i < 3; $i++){
        $data = array_map('intval', explode(' ', fgets(STDIN)));
        $hour = $data[0];
        $minutes = $data[1];
        $time = "AM";
        if($hour >= 13){
            $hour -= 12;
            $time = "PM";
        }
        if($hour == 12){
            $time = "PM";
        }
        if($hour == 0){
            $hour = 12;
        }
        $count += ($time == "AM" ? 1 : 0);
        printf("%s %d:%d\n", $time, $hour, $minutes);
    }
    printf("%d\n", $count);
?>